src/main/resources/org/jenkinsci/plugins/ewm/actions/ExwsAllocateActionWrapper/index.jelly

```diff
@@ -0,0 +1,11 @@
+<?jelly escape-by-default='true'?>
+<st:compress xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
+    <l:layout title="${it.displayName}" norefresh="true">
+        <st:include it="${it.parent}" page="sidepanel.jelly"/>
+        <l:main-panel>
+            <j:forEach var="exwsAction" items="${it.allActions}">
+                <st:include it="${exwsAction}" page="index.jelly"/>
+            </j:forEach>
+        </l:main-panel>
+    </l:layout>
+</st:compress>
```

Comment:
In one of these jelly files, does it make sense to display to the user, that when multiple allocated workspace have been allocated, only the first one will be used by downstream jobs? Is this a good place to remind the user of this information?